app/views/static/_home_sections.html.erb

```diff
@@ -0,0 +1,84 @@
+<section class="home-section home-section--gray">
+  <div class="container">
+    <div class="row">
+      <div class="col-xs-12 col-sm-5">
+        <div class="home-section__thumb">
+          <%= image_tag 'static/home-screenshot-01.jpg' %>
+        </div>
+      </div>
+      <div class="col-xs-12 col-sm-7">
+        <h2 class="home-section__text">
+          <small>Section One</small>
+          Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat
+        </h2>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="home-section">
+  <div class="container">
+    <div class="row">
+      <div class="col-xs-12 col-sm-5 col-sm-push-7">
+        <div class="home-section__thumb">
+          <%= image_tag 'static/home-screenshot-02.jpg' %>
+        </div>
+      </div>
+      <div class="col-xs-12 col-sm-7 col-sm-pull-5">
+        <h2 class="home-section__text">
+          <small>Section Two</small>
+          Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat
+        </h2>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="home-section home-section--gray home-section--form">
+  <div class="container">
+    <div class="row">
+      <div class="col-xs-12 col-sm-6 col-sm-offset-3">
+        <h3 class="signup-text signup-text--bottom text-center">Start learning now</h3>
+        <form class="form-horizontal signup-form signup-form--bottom">
```

Comment:
This form should probably be copied from the devise registration file which already works. https://github.com/eruditionapp/eruditionapp/blob/master/app/views/devise/registrations/new.html.erb